src/utils/Utils.cc

```diff
@@ -425,7 +425,7 @@ namespace pfs { namespace drp { namespace stella { namespace utils{
     double goodVal = yArr[badPos];
     yArr[badPos] = badVal;
 
-    /// Test PolyFit(xArr, yArr, nDeg)
+    cout << "Test PolyFit(xArr, yArr, nDeg)" << endl;
```

Comment:
I think the approach you are taking here is fundamentally wrong-headed.

You have a single giant function which tests a number of different calls to `PolyFit`. Rather than doing that, you ought to have a number of short unit tests, each of which tests a single call.

Ideally, each of the unit tests would be written in Python. However, I understand that the full `PolyFit` interface is not exposed to Python, so this is impractical. That leaves you with two choices:

- Use a C++-based test harness. The one in use in LSST is [Boost::Test](http://www.boost.org/doc/libs/1_63_0/libs/test/doc/html/index.html). For example, see [these tests in afw](https://github.com/lsst/afw/blob/a88e8e0d1ec5b1343fb138d9ed0cfbf5e923b889/tests/footprintArray.cc#L56).
- Write individual test functions in C++ then expose each one to Python by wrapping it separately using Swig.

Either of those approaches is adequate. What you've got now isn't.

However, I don't think you need to address that on this ticket. Therefore, I am minded to temporarily accept what you've done here, on the condition that you file another ticket to come back and clean up properly sometime in future.